docs/src/getting-started/setup.md

```diff
@@ -0,0 +1,13 @@
+# Setup instructions for the Fuel SDK
+
+
+## What you will need on your machine
+
+- Rust 2021 ([see here](https://doc.rust-lang.org/cargo/getting-started/installation.html))
+- A clone of the `fuels-rs` repository: 
+```
+git clone https://github.com/FuelLabs/fuels-rs
+```
+
+That's it! Thanks to the awesome `cargo` package management, you don't need to install the Fuel 
+node or Sway yourself.
```

Comment:
Wait no, I'm pretty sure you still need to install `forc` and `fuel-core`, no 😂?
```suggestion
```

Reply:
I wasn't sure how to test this but you are probably right haha